src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,52 @@
+.top-bar {
+  padding-block: 24px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @include on-tablet {
+    padding-block: 36px;
+  }
+
+  &__icons {
+    display: flex;
+    gap: 24px;
+  }
+
+  &__phone-number {
+    visibility: hidden;
```

Comment:
The `visibility: hidden` property cannot be animated, so consider using `opacity` for smooth transitions instead.
Fix in all such cases 🥹